doc/quickstart/intro.rst

```diff
@@ -0,0 +1,151 @@
+ .. role:: html(raw)
+   :format: html
+
+.. _pl_intro:
+
+Introduction
+============
+
+This section is an introduction to how the concept of a :ref:`variational quantum circuit <varcirc>`
+is implemented in PennyLane.
+
+It shows new PennyLane users how to:
+
+.. image:: ../_static/intro_qnodes.png
+    :align: right
+    :width: 180px
+    :target: javascript:void(0);
+
+- Construct quantum circuits via **quantum functions**
+- Define **computational devices**
+- Combine quantum functions and devices to **quantum nodes**
+- Conveniently create quantum nodes using the quantum node **decorator**
+- Find out more about **interfaces** to use quantum nodes
```

Comment:
Find out more about **interfaces** to use with quantum nodes

Reply:
```suggestion
- Find out more about **interfaces** to use quantum nodes
```